files/zh-cn/web/javascript/reference/global_objects/array/with/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Array.prototype.with()
+slug: Web/JavaScript/Reference/Global_Objects/Array/with
+page-type: javascript-instance-method
+browser-compat: javascript.builtins.Array.with
```

Comment:
```suggestion
```

remove according to [Translation-Guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)